charts/sourcegraph/templates/pgsql/pgsql.ConfigMap.yaml

```diff
@@ -1,3 +1,6 @@
+{{- if and .Values.pgsql.enabled (and .Values.pgsql.existingConfig .Values.pgsql.additionalConfig) -}}
+{{- fail "You can only define one of 'pgsql.existingConfig' and 'pgsql.additionalConfig' at a time" }}
+{{- end }}
```

Comment:
I think this is valuable cuz defining both `existingConfig` and `additionalConfig` could cause confusion where only `existingConfig` will be used. It's better to fail instead of "work" sliently